scst-store/deployment-details.hbs.md

```diff
@@ -41,7 +41,7 @@ For a production environment, VMware recommends installing SCST - Store with ing
 
 #### <a id='amr'></a>Deploying AMR
 
->**Note** AMR is an alpha feature and is not recommended for use in production.
+>**Note** AMR is a beta feature and is not recommended for use in production.
 
 By default, AMR is not deployed with SCST Store. There is an `amr` section inside `metadata-store`. To deploy AMR, you must set the `deploy` property under `amr` to `true`.
```

Comment:
this is the field that is in the tap-values yaml, so it's supposed to be `metadata_store`. But actually I see there is a typo there, it should be `_` instead of `-`, I'll change that